.travis.yml

```diff
@@ -74,6 +74,6 @@ script:
         --rm \
         --workdir /travis \
         --volume ${TRAVIS_BUILD_DIR}:/travis \
-        debian:sid /bin/bash -c './debian-setup.sh && ./driver.sh && ccache -s'
+        nathanchance/cbl /bin/bash -c './ccache-setup.sh && ./driver.sh && ccache -s'
```

Comment:
I think it would be good to move the `cbl` image to something from our org.

Reply:
Done.